infra/base-images/base-runner/install_javascript.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -eux
```

Comment:
@oliverchang What do you think of moving this and install_java and install_go into base-image so we don't have to duplicate them?
(I guess this can come in a new PR).